doc/shared/sundials/Install.rst

```diff
@@ -1761,6 +1768,13 @@ configuration file to build against SUNDIALS in their own CMake project.
   target_link_libraries(myexec PUBLIC SUNDIALS::cvode SUNDIALS::nvecpetsc)
 
 
+.. note::
+
+   Starting with SUNDIALS v7.1.0, a single version provided to ``find_package``
+   denotes the minimum version of SUNDIALS to look for, and any version equal
+   or newer than what is specified will match.
```

Comment:
```suggestion
   .. versionchanged:: x.y.z

      A single version provided to ``find_package`` denotes the minimum version
      of SUNDIALS to look for, and any version equal or newer than what is
      specified will match. In prior versions ``SUNDIALSConfig.cmake`` required
      the version found to have the same major version number as the single
      version provided to ``find_package``.
```

Reply:
Question about leaving this as `x.y.z`, will the version update script find these and replace them? Or do we have to notice it in the release PR?

Reply:
The version update script will find and replace them.